### PR TITLE
Iguk 314 buy from the uk contact

### PR DESCRIPTION
**Pull request by stuart-mindt**
Iguk 314 buy from the uk contact

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/IGUK-314
- [x] Jira ticket has the correct status.
- [x] A clear/description pull request messaged added.

### Reviewing help

- [x] Explains how to test locally, including how to set up appropriate data
- [x] Includes screenshot(s) - ideally before and after, but at least after
image

### Housekeeping

- [x] Frontend assets have been re-compiled.

### Merging

- [ ] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: international_buy_from_the_uk/templates/buy_from_the_uk/contact.html [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+            <div class="govuk-grid-column-one-half-from-desktop">
+                <form method="post" onsubmit="return onSubmitContact()">
+                    {% csrf_token %}
+                    <div class="govuk-grid-row govuk-!-margin-bottom-4">
+                        <div class="govuk-grid-column-full">
+                            <div class="{% if form.given_name.errors %} govuk-form-group--error{% endif %}">
+                                {% include 'international/includes/form_field.html' with field=form.given_name %}
+                            </div>
+                        </div>
+                    </div>
```
Comment (timothyPatterson) [inline international_buy_from_the_uk/templates/buy_from_the_uk/contact.html:42]: If you have appetite to change this pattern can be replaced with `govuk-form-group` see [here](https://uktrade.atlassian.net/browse/IGUK-300). Also `international/includes/form_field.html` now accepts a parameter `show_vertical_error_line`, e.g. implementation [here](https://github.com/uktrade/great-cms/blob/5e13dcd41601c06981c9ac7abe48582f8e3c32a6/international_online_offer/templates/eyb/triage/business_details.html#L32).

File: international_buy_from_the_uk/templates/buy_from_the_uk/contact.html [hunk 2]
```diff
@@ -0,0 +1,548 @@
...
+        const onSubmitContact = () => {
+            if (!document.getElementById('js-industry-select').value) {
+                document.getElementById('js-industry-select-select').value = '';
+            }
+            if (!document.getElementById('js-country-select').value) {
+                document.getElementById('js-country-select-select').value = '';
+            }
+        }
+
+        const countries = [
```
Comment (timothyPatterson) [inline international_buy_from_the_uk/templates/buy_from_the_uk/contact.html:196]: Could this list be imported from somewhere? realising it is now in a few places.

**Review by timothyPatterson — APPROVED** [review]
Couple of small comments but realise this is a lift and shift aiming at feature parity.

File: international_buy_from_the_uk/templates/buy_from_the_uk/contact.html (hunk 2 above)
Comment (stuart-mindt) [reply international_buy_from_the_uk/templates/buy_from_the_uk/contact.html]: Done. I already did this and put it into a poorly named file in core. Renamed the file as its just countries in JS form for others to use and imported into places we use it across international

File: international_buy_from_the_uk/templates/buy_from_the_uk/contact.html (hunk 1 above)
Comment (stuart-mindt) [reply international_buy_from_the_uk/templates/buy_from_the_uk/contact.html:42]: Bigger piece to do here around only having one form_field in international. Currently we have one in international and one in international_online_offer. Pretty sure they're the same but just need to go through and update the imports then test all is working as expected. Will pickup in the next one :)

Comment (stuart-mindt) [reply international_buy_from_the_uk/templates/buy_from_the_uk/contact.html:42]: Did this in the end. Was fairly simple

**Pull request merged**